salt/salt.spec

```
@@ -789,7 +789,16 @@ Requires:       iputils
 Requires:       sudo
 Requires:       file
 Recommends:     man
+%if 0%{?rhel} || 0%{?fedora}
 Recommends:     python3-passlib
+%endif
+%if 0%{?suse_version}
+%if 0%{?singlespec_compat}
+Recommends:     %{python_module passlib}
+%else
+Recommends:     python-passlib
+%endif
+%endif
```

This part seems overcomplicated, for `0%{?singlespec_compat}` case `%{python_module passlib}` should produce exactly `python3-passlib` as there is `%define python_module() python3-%**` set for such case already. So for me looks like the following should give exactly the same result:
```suggestion
%if 0%{?singlespec_compat}
Recommends:     %{python_module passlib}
%else
Recommends:     python-passlib
%endif
```

---

You are right. Thanks for the suggestion.